src/types/streaming_server/settings.rs

```diff
@@ -7,6 +7,7 @@ pub struct Settings {
     pub cache_root: String,
     pub server_version: String,
     pub remote_https: Option<String>,
+    pub transcode_profile: Option<String>,
```

Comment:
Please document what this value is used for and some examples of what it can include